### ISSUE TITLE
Update for iron 0.6.x

### ISSUE THREAD

**Issue by BransonGitomeh** [iron/params#37]
Thanks for this awesome package 

But im getting the errors
```
error[E0277]: the trait bound `params::Params: plugin::Plugin<iron::Request<'_, '_>>` is not satisfied
  --> src\main.rs:40:23
   |
40 |         let map = req.get_ref::<Params>().unwrap();
   |                       ^^^^^^^ the trait `plugin::Plugin<iron::Request<'_, '_>>` is not implemented for `params::Params`
   |
   = help: the following implementations were found:
             <params::Params as plugin::Plugin<iron::request::Request<'a, 'b>>>
```

on checking my cargo.toml

```toml
[[package]]
name = "params"
version = "0.7.1"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "bodyparser 0.6.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "iron 0.5.1 (registry+https://github.com/rust-lang/crates.io-index)",
 "multipart 0.12.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "num 0.1.40 (registry+https://github.com/rust-lang/crates.io-index)",
 "plugin 0.2.6 (registry+https://github.com/rust-lang/crates.io-index)",
 "serde_json 0.9.10 (registry+https://github.com/rust-lang/crates.io-index)",
 "tempdir 0.3.5 (registry+https://github.com/rust-lang/crates.io-index)",
 "urlencoded 0.5.0 (registry+https://github.com/rust-lang/crates.io-index)",
]
```

I cant downgrade to iron 0.5  since it breaks other packages

This is for the simple code on your readme
```rust
fn handle_user(req: &mut Request) -> IronResult<Response> {
        use params::{Params, Value};
        let map = req.get_ref::<Params>().unwrap();

        match map.find(&["user", "name"]) {
            Some(&Value::String(ref name)) if name == "Marie" => {
                Ok(Response::with((iron::status::Ok, "Welcome back, Marie!")))
            },
            _ => Ok(Response::with(iron::status::NotFound)),
        }
    }
```

**Comment by birktj** [comment]
I have the same problem, a quick fix is to simply to clone this repository and update the dependencies in the Cargo.toml file.

**Comment by SkylerLipthay** [comment]
Closed by #39.

**Comment by BransonGitomeh** [comment]
thank you @SkylerLipthay
